firebase-firestore/src/main/java/com/google/firebase/firestore/local/IndexManager.java

```diff
@@ -32,6 +32,21 @@
  * Collection Group queries.
  */
 public interface IndexManager {
+  /** Represents the index state as it relates to a particular target. */
+  public enum IndexStatus {
```

Comment:
I don't have a suggestion, but we should see if there is a different name since we already have IndexState which stores how up to date an index is.